schema/places/place.yaml

```diff
@@ -21,7 +21,7 @@ properties:
     allOf:
       - "$ref": ../defs.yaml#/$defs/propertyContainers/overtureFeaturePropertiesContainer
     properties:
-      names: { "$ref": "../defs.yaml#/$defs/propertyDefinitions/names" }
+      names: { "$ref": "../defs.yaml#/$defs/propertyDefinitions/namesv2" }
```

Comment:
Any reason to retain the old names at this point?

It seems like by doing so and adding `namesv2` we're just creating an opportunity for tech debt/cruft to build up, since there will be an outstanding action to delete the old names stuff and rename `namesv2` to `names`. Might we as well not just do that now?

Reply:
Done.